From 3819a3822af62c56c0e1f4b0ab1c6aea9ae147bb Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Thu, 17 Apr 2014 12:12:29 -0700
Subject: [PATCH 95/99] tests: t1700 change count to seek

Fix a typo when changing to sparse images.

* tests/t1700-probe-fs.sh: count should be seek.
---
 tests/t1700-probe-fs.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/t1700-probe-fs.sh b/tests/t1700-probe-fs.sh
index 08ec7d9..bf6ad9c 100755
--- a/tests/t1700-probe-fs.sh
+++ b/tests/t1700-probe-fs.sh
@@ -33,7 +33,7 @@ for type in ext2 ext3 ext4 btrfs xfs nilfs2; do
       *) force=;; esac
 
   # create an $type file system
-  dd if=/dev/null of=$dev bs=$ss count=$n_sectors >/dev/null || fail=1
+  dd if=/dev/null of=$dev bs=$ss seek=$n_sectors >/dev/null || fail=1
   mkfs.$type $force $dev || { warn_ $ME: mkfs.$type failed; fail=1; continue; }
 
   # probe the $type file system
@@ -45,7 +45,7 @@ done
 # Some features should indicate ext4 by themselves.
 for feature in uninit_bg flex_bg; do
   # create an ext3 file system
-  dd if=/dev/null of=$dev bs=1024 count=4096 >/dev/null || fail=1
+  dd if=/dev/null of=$dev bs=1024 seek=4096 >/dev/null || fail=1
   mkfs.ext3 -F $dev >/dev/null || skip_ "mkfs.ext3 failed"
 
   # set the feature
-- 
1.9.0

